ros/src/tl_detector/tl_detector.py

```diff
@@ -10,18 +10,33 @@
 import tf
 import cv2
 import yaml
+import math
+import time
 
 STATE_COUNT_THRESHOLD = 3
+GROUND_TRUTH_PASS = True  
+#Variable to use ground truth data until the classifier has been developed
 
 class TLDetector(object):
     def __init__(self):
         rospy.init_node('tl_detector')
 
+		#Private Parameter to differentiate simulation or Site Launch  
+		#Default - site launch
+        self.isSimulation = bool(rospy.get_param('~isSimulation', False))
```

Comment:
Is this flag used somewhere?

Reply:
I updated the algorithm after debug- but forgot to clean this up. Will update the branch with this fixed.